README.md

```diff
@@ -63,11 +61,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Vet Clinic] <a name="about-project"></a>
```

Comment:
- Please fill missing information in the readme file. Add the right information for the tech stack used, add explicit commands for getting started. How to clone, setup, install